programming/sr/ruggeduinos/custom_firmware.md

```diff
@@ -25,6 +25,11 @@ If you're using completely custom firmware, you'll need to tell the kit to ignor
 [Extension of the SR firmware](#extension) {#extension}
 ------------------------------
 
+<div class="warning">
```

Comment:
To enable markdown parsing inside the HTML block you'll need to annotate it as such:
```suggestion
<div class="warning" markdown="1">
```